src/js/profile/mobile/widget/DropdownMenu.js

```diff
@@ -1076,7 +1084,10 @@
 					ui.elOptionWrapper.classList.add(classes.verticalMargins);
 				}
 
-				optionStyle += "top: " + offsetTop + "px; width: " + width + "px; max-height: " + height + "px;";
+				optionStyle += "top: " + offsetTop + "px; max-height: " + height + "px;";
+				if (options.inline === true) {
```

Comment:
Not sure if this option is still supported and works as it should be. Are we sure that width should be excluded  one inline option is specified?

Reply:
You are right, `inline` change is out of scope this issue.